addOns/custompayloads/src/main/resources/org/zaproxy/zap/extension/custompayloads/resources/Messages.properties

```diff
@@ -36,6 +36,7 @@ custompayloads.options.dialog.addMultiplePayload.error.title = Error Adding Payl
 custompayloads.options.dialog.addMultiplePayload.selectFile.button.name = Select File
 custompayloads.options.dialog.addMultiplePayload.title = Add Multiple Payloads
 custompayloads.options.dialog.category = Category
+custompayloads.options.dialog.disabled = This functionality is currently disabled. Added payloads will not be saved.
```

Comment:
We should tell why it's disabled. (The latter part will not be possible with the other method call.)